pinot-spi/src/main/java/org/apache/pinot/spi/ingestion/batch/BatchConfigProperties.java

```diff
@@ -43,7 +43,9 @@ private BatchConfigProperties() {
   public static final String TABLE_CONFIGS_URI = "tableConfigsURI";
   public static final String SCHEMA = "schema";
   public static final String SCHEMA_URI = "schemaURI";
+  public static final String UPLOADED_REALTIME_PARTITION_ID = "uploadedRealtimePartitionId";
```

Comment:
nit: just call it `segmentPartitionId`?

per the naming convention here, perhaps better to name them like below:
```
segment.partitionId
segment.uploadTimeMs
```